### ISSUE TITLE
Document supported hardware architectures

### ISSUE THREAD

**Issue by zamazan4ik** [google/leveldb#1035]
Hi!

I have checked the documentation but did not find any word about supported architectures - only about supported operating systems.

Could you please put the information about supported architectures to the documentation please? E.e. about supported architectures for different operating systems, some specific requirements to the supported instructions, if you have any (e.g. maybe AVX is required - I do not know).

This kind of information is important for the end-users.

Thanks in advance!

**Comment by zamazan4ik** [comment]
Ok. For which OS-Arch combinations do you have a CI? I guess some architectures are more tested than others.

**Comment by zamazan4ik** [comment]
Continuous Integration

**Comment by zamazan4ik** [comment]
So could I interpret it as "Only x86-64 architecture is tested"? Since Github Actions works only on x86-64 builders (without self-hosted workers and/or QEMU-based magic).

**Comment by cmumford** [comment]
Yes - for the official CI builders. Other projects, like Google Chrome, have an expanded set of builders that run on ARM (for Android and Chrome OS), but it's the responsibility of those downstream consumers to deal with any architecture issues.
